lib/matplotlib/font_manager.py

```diff
@@ -380,6 +382,22 @@ def findSystemFonts(fontpaths=None, fontext='ttf'):
     return [fname for fname in fontfiles if os.path.exists(fname)]
 
 
+def fontentry_helper_repr_png_(fontent):
+    from matplotlib.figure import Figure  # Circular import.
+    fig = Figure()
+    font_path = Path(fontent.fname) if fontent.fname != '' else None
+    fig.text(0, 0, fontent.name, font=font_path)
+    with BytesIO() as buf:
+        fig.savefig(buf, bbox_inches='tight', transparent=True)
```

Comment:
Is this enough to get high-dpi or does this just make things bigger?

```suggestion
        fig.savefig(buf, bbox_inches='tight', transparent=True, dpi=200)
```

Reply:
That will just make it bigger. The HTML needs to specify that the image is 2x, and I don't know how to tell that to Jupyter via the `_repr_png_` interface.